torch/quantization/fx/prepare.py

```diff
@@ -458,7 +461,19 @@ def maybe_insert_output_observer_for_node(
                     get_default_output_activation_post_process_map().get(
                         matched_pattern,
                         act_post_process_ctr)
+
             observer = act_post_process_ctr()
+            if isinstance(observer, _InputEqualizationObserver):
```

Comment:
we should only insert output observers inside of the `maybe_insert_output_observer_for_node`.  If I understand this PR correctly, a better place for inserting the equalization input observer object would be somewhere in `maybe_insert_input_observers_for_node`.